tests/components/button.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Button from "@libs/components/button";
+
+describe("components/Button", () => {
+  it("can be childrened", () => {
+    const component = renderer.create(<Button children="Click me" />).toJSON();
```

Comment:
that's not how we pass `children` prop in React. This is actually a special prop that means "things inside the component":

```js
<Button>
  children are here!
</Button>
```